packages/babel-plugin-transform-async-to-generator/test/fixtures/regression/4943/options.json

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": ["es2015"],
```

Comment:
I didn't realize this should be moved out for 7.0 branch, ill go ahead and do that.